examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,75 @@
+import { Reach, test } from '@reach-sh/stdlib';
+const stdlib = new Reach(process.env, { REACH_NO_WARN: 'Y' });
+
+const startingBalance = stdlib.parseCurrency(100);
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+
+const gasLimit = 500000;
+if (stdlib.connector != 'ALGO') {minter.setGasLimit(gasLimit)};
+if (stdlib.connector != 'ALGO') {receiver.setGasLimit(gasLimit)};
+
+const mintAddr = minter.getAddress();
+console.log(`Minter's address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`Receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who, tok) => fmt(await stdlib.balanceOf(who, tok));
+
+const logBalance = async (acc, tok, accStr) => {
+    const bal = await getBal(acc, tok, accStr);
+    const trueBal = await tok ? opts.supply : bal;
+    const unit = await tok ? 'of the NFT' : stdlib.connector;
```

Comment:
TIL of `stdlib.standardUnit`. It should be used instead of `stdlib.connector`. Right now, `standardUnit == connector` for all connectors, but that might not be the case for future connectors and standard unit is semantically correct. Also, this doesn't need to be awaited